CHANGELOG.md

```diff
@@ -16,7 +16,7 @@
 ### Backward-incompatible changes [experimental]
 
 ### Performance enhancements
-* `skbio.tree.nj` wall-clock time was decreased about 66% by vectorizing `skbio.tree._nj._compute_q`. ([#1512](https://github.com/biocore/scikit-bio/pull/1512))
+* `skbio.tree.nj` wall-clock time was decreased about 99% by vectorizing `skbio.tree._nj._compute_q` and avoiding unnecessary recalculations. ([#1512](https://github.com/biocore/scikit-bio/pull/1512), [#1513](https://github.com/biocore/scikit-bio/pull/1513))
```

Comment:
Ah, sorry, I "approved" the changes before the matrix size inclusion :) This is good to merge in my opinion once that's in.